lib/navigation/benchmark/benchmark.hpp

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+#include <functional>
+#include <map>
+#include <memory>
+#include <vector>
+
+#include <core/time.hpp>
+#include <navigation/navigator.hpp>
+#include <utils/manual_time.hpp>
+
+namespace hyped::navigation::benchmark {
+
+struct TrajectoryError {
+  const core::Float displacement;
+  const core::Float velocity;
+  const core::Float acceleration;
+
+  TrajectoryError(const core::Trajectory &expected, const core::Trajectory &actual);
+};
+
+struct Result {
```

Comment:
Should this not go in Navigation/consts.hpp?

Reply:
Or the new types header?

Reply:
It should definitely stay here because it's a benchmark result (`navigation::benchmark::Result`)